lib/turn.rb

```diff
@@ -0,0 +1,35 @@
+class Turn
+  GAME_RULES = {
+    rock: { rock: :draw, paper: :lose, scissors: :win },
+    paper: { rock: :win, paper: :draw, scissors: :lose },
+    scissors: { rock: :lose, paper: :win, scissors: :draw }
+  }
```

Comment:
This is a clever way of writing the rules out, which means you can make the function below private.